src/env_properties.h

```diff
@@ -215,6 +215,7 @@
   V(iterator_string, "Iterator")                                               \
   V(jwk_crv_string, "crv")                                                     \
   V(jwk_d_string, "d")                                                         \
+  V(database_string, "database")                                               \
```

Comment:
That seems out of order, AKAICT the rest of the list is ordered by the key, that should be moved to line 120